clients/python/girder_client/__init__.py

```diff
@@ -706,7 +706,8 @@ def createCollection(self, name, description='', public=False):
         }
         return self.createResource('collection', params)
 
-    def createFolder(self, parentId, name, description='', parentType='folder', public=None):
+    def createFolder(self, parentId, name, description='', parentType='folder',
+                     reuseExisting=False, public=None):
```

Comment:
`reuseExisting` should be tacked on to the end to preserve backwards compatibility for anyone already using it.